.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Report a bug, false-positive or false-negative
+title: '[BUG] '
```

Comment:
I don't know if you want the issues to have some specific format.

Reply:
I'm not a big fan of `[label]` in the title. Then we have to go through and edit them.